client/src/about/components/About.tsx

```diff
@@ -2,9 +2,7 @@ import React, { FC } from 'react';
 import './About.scss';
 
 const About: FC = () => (
-  <div className='about-page'>
-    <h1>OwlPlace</h1>
-
+  <div className="about-page">
```

Comment:
bruh... we want single quotes

Reply:
bruh...prettier stopped support for vscode, so I had to do stuff with package.json